src/types/display_data_address.rs

```diff
@@ -1,71 +1,67 @@
-use bitflags::bitflags;
 use core::fmt;
+use num_enum::{IntoPrimitive, TryFromPrimitive};
 
-bitflags! {
-    /// Display RAM data address.
-    pub struct DisplayDataAddress: u8 {
-        /// Row 0
-        const ROW_0 = 0;
-        /// Row 1
-        const ROW_1 = 1;
-        /// Row 2
-        const ROW_2 = 2;
-        /// Row 3
-        const ROW_3 = 3;
-        /// Row 4
-        const ROW_4 = 4;
-        /// Row 5
-        const ROW_5 = 5;
-        /// Row 6
-        const ROW_6 = 6;
-        /// Row 7
-        const ROW_7 = 7;
-        /// Row 8
-        const ROW_8 = 8;
-        /// Row 9
-        const ROW_9 = 9;
-        /// Row 10
-        const ROW_10 = 10;
-        /// Row 11
-        const ROW_11 = 11;
-        /// Row 12
-        const ROW_12 = 12;
-        /// Row 13
-        const ROW_13 = 13;
-        /// Row 14
-        const ROW_14 = 14;
-        /// Row 15
-        const ROW_15 = 15;
+use DisplayDataAddress::*;
```

Comment:
Changing from bitflags into enum as it makes more sense. Addresses should not need bitwise operations.